kep/61-real-load-aware-scheduling/61-real-load-aware-scheduling.md

```diff
@@ -87,7 +88,10 @@ The file will be stored in host file system, so it will be persisted across pod
 
 ### Scheduler Plugins
 
-This uses the scheduler framework of K8s to incorporate our custom real load aware scheduler plugins without modifying the core scheduler code.
+This uses the scheduler framework of K8s to incorporate our customized real load aware scheduler plugins without modifying the core scheduler code. The plugins we proposed mainly include the following two.
```

Comment:
This is for discussion: https://github.com/kubernetes-sigs/scheduler-plugins/pull/61#discussion_r502605840